OMX-27-firmware/sequencer.cpp

```diff
@@ -9,7 +9,7 @@
 
 // globals in main ino
 extern SequencerState sequencer;
-// extern OMXMode omxMode;
+extern SysSettings sysSettings;
```

Comment:
Long-term, ideally all sequencer functionality would become encapsulated in a `SequencerMode` class or the like, and we'd inject the settings (and other dependencies) into it's constructor.